pkg/networkservice/common/excludedprefixes/utils.go

```diff
@@ -31,3 +32,20 @@ func removeDuplicates(elements []string) []string {
 	}
 	return result
 }
+
+// Exclude - excludes strings from a slice
+func Exclude(source, exclude []string) []string {
+	var prefix string
+
+	for i := 0; i < len(source); i++ {
+		prefix = source[i]
+		for _, ipCtxPrefix := range exclude {
+			if prefix == ipCtxPrefix {
+				source = append(source[:i], source[i+1:]...)
+				i--
+			}
+		}
+	}
```

Comment:
Please use the `map[string]struct{}` and reduce the time complexity from O(n*m) to O(n + m)

Reply:
Done